rkt/prepare.go

```diff
@@ -75,7 +75,11 @@ func runPrepare(cmd *cobra.Command, args []string) (exit int) {
 		}
 	}
 
-	if flagPrivateUsers && common.SupportsUserNS() {
+	if flagPrivateUsers {
+		if !common.SupportsUserNS() {
+			stderr("run: --private-users is not supported, kernel compiled without user namespace support.")
```

Comment:
To be consistent with other `stderr()` calls in this file, you can remove the final dot (`.`).